src/main/java/com/spikes2212/control/JoinedPID.java

```diff
@@ -0,0 +1,42 @@
+package com.spikes2212.control;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class JoinedPID implements  PIDLoop{
+
+    private List<PIDLoop> pidLoops;
+
+    public JoinedPID(PIDLoop... pidLoops){
+        this.pidLoops.addAll(Arrays.asList(pidLoops));
```

Comment:
you can remove `this` from here.

Reply:
doing so would make pidLoops to be the given variable and not the class' variable